packages/rpc-provider/src/ws/send.spec.ts

```diff
@@ -31,7 +34,6 @@ describe('send', () => {
 
     if (mock) {
       mock.done();
-      mock = null;
     }
   });
 
```

Comment:
Here I could also need a second pair of eyes.
I'm getting this error with my `Global` solution:

```
packages/rpc-provider/src/ws/send.spec.ts:45:5 - error TS2740: Type 'typeof WebSocket' is missing the following properties from type 'WebSocket': binaryType, bufferedAmount, extensions, onclose, and 15 more.

45     global.WebSocket = class {
```

and 

```

packages/rpc-provider/src/ws/send.spec.ts:56:5 - error TS2532: Object is possibly 'undefined'.

56     websocket.onopen();
```

Reply:
global.WebSocket = class extends WebSocket

Reply:
Closed because it's a duplicate of https://github.com/polkadot-js/api/pull/936#discussion_r287268555